src/modules/seventv/utils.js

```diff
@@ -24,6 +24,9 @@ export function createEmote(id, code, animated, owner, category) {
       '2x_static': animated ? emoteUrl(id, '2x', true) : undefined,
       '4x_static': animated ? emoteUrl(id, '4x', true) : undefined,
     },
+    metadata: {
+      isOverlay: hasFlag(flags, 1 << 0),
```

Comment:
I believe the overlay flag should be `1 << 8` (256), not `1 << 0` (1)

Reply:
this comment is correct, the overlay flag appears to be `1 << 8`.